lib/puppetserver/ca/action/setup.rb

```diff
@@ -77,8 +77,8 @@ def generate_pki(settings, signing_digest)
           ca = Puppetserver::Ca::LocalCertificateAuthority.new(signing_digest, settings)
 
           root_key, root_cert, root_crl = ca.create_root_cert
-          int_key, int_cert, int_crl = ca.create_intermediate_cert(root_key, root_cert)
-          master_key, master_cert = ca.create_master_cert(int_key, int_cert)
+          ca.create_intermediate_cert(root_key, root_cert)
```

Comment:
God, I want to write something really Java-y here, like this could be a custom constructor:
```rb
ca = LocalCertificateAuthority.asIntermediateFrom(root_key, root_cert)
```

And I think the gist of that is applicable, but I'm failing to think of an idiomatic way call for it. Anyways, another nitpicky idea.